libs/core/src/lib/busy-indicator/busy-indicator.component.scss

```diff
@@ -1,21 +1,25 @@
 @import 'fundamental-styles/dist/busy-indicator';
 
-.fd-busy-indicator {
+* + .fd-busy-indicator,
+* + .fd-busy-indicator-extended__label {
+    white-space: nowrap;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+* + .fd-busy-indicator-extended__label {
```

Comment:
Same thing with this. This style will always be applied because it will always have a sibling before